targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala

```diff
@@ -167,36 +159,32 @@ object TargetBody {
                         clazz = ExploreStyles.FormSectionLabel |+| ExploreStyles.ColumnSpan(2)
                   ),
                   InputWithUnits(
-                    props.target.zoom(TargetQueries.pvRALens),
+                    props.target.zoom(TargetQueries.pvRALens).withOnMod(modifyProperVelocitRA),
```

Comment:
This makes the mutation part of the `View` setter and I think is the pattern we should use. `FormInputEV` will only call the whole thing if the value changed, whereas `onBlur` is always called.